safedelete/queryset.py

```diff
@@ -90,10 +91,12 @@ def _filter_visibility(self):
         This is because QuerySet._fetch_all cannot work with a clone.
         """
         force_visibility = getattr(self, '_safedelete_force_visibility', None)
+        filter_applied = getattr(self, '_safedelete_filter_applied', False)
```

Comment:
Yes you are right, I was before doing a safe get attr but that changed, let me update that and push